com.unity.ml-agents/Tests/Runtime/CameraSensorTest.cs

```diff
@@ -0,0 +1,69 @@
+ using System;
+ using System.Collections;
+ using NUnit.Framework;
+ #if UNITY_EDITOR
+ using UnityEditor.SceneManagement;
+ using UnityEngine.TestTools;
+ #endif
+ using UnityEngine;
+ using UnityEngine.SceneManagement;
+ using MLAgents;
+
+namespace MLAgents.Tests
+{
+
+    [TestFixture]
+    public class CameraSensorTest
+    {
+        private class CameraTestAgent : Agent
+        {
+            public override void CollectObservations(VectorSensor sensor)
```

Comment:
Can you just not implement this method?

Reply:
And step the sensors there? Might be OK for testing, but it's not something a user would be able to do if they wanted to inspect the sensor output.